ui-tests/src/test/java/io/syndesis/qe/steps/other/DocsVerificationSteps.java

```diff
@@ -0,0 +1,50 @@
+package io.syndesis.qe.steps.other;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import static com.codeborne.selenide.Condition.visible;
+import static com.codeborne.selenide.Selenide.$;
+import static com.codeborne.selenide.WebDriverRunner.url;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.firefox.FirefoxDriver;
+
+import com.codeborne.selenide.Selenide;
+
+import java.util.HashMap;
+import java.util.concurrent.TimeUnit;
+
+import cucumber.api.java.en.Then;
+
+public class DocsVerificationSteps {
+    private HashMap<String, String> versions = new HashMap<>();
+    private static String versionOnAboutPage;
+
+    public DocsVerificationSteps() {
+        versions.put("1.9", "7.6");
+    }
+
+    @Then("^check version in about page$")
+    public void checkVersion() {
+        final By SYNDESIS_VERSION = By.cssSelector("[data-testid=\"about-modal-content-version-list-item\"]");
```

Comment:
if this variable should be a constant, it should be defined as `private static final By SYNDESIS_VERSION` on class level, if not, then it should be named `syndesisVersion`